cypress/support/commands.js

```diff
@@ -23,3 +23,17 @@
 //
 // -- This will overwrite an existing command --
 // Cypress.Commands.overwrite('visit', (originalFn, url, options) => { ... })
+
+Cypress.Commands.add('login', () => {
+  const userName = 'Anton3108';
+  const password = 'A12345678a!';
+cy.request('POST', 'https://demoqa.com/Account/v1/Login', {
+  userName: userName,
+  password: password
+}).then(response => {
+  cy.setCookie('token', response.body.token);
+  cy.setCookie('userID', response.body.userId);
+  cy.setCookie('expires', response.body.expires);
+
+});
+});
```

Comment:
```suggestion
});

```

Reply:
Not fixed.
Add empty line into the end of the file